synapse/lib/modelrev.py

```diff
@@ -759,52 +759,50 @@ async def _runStorm():
 
     async def revCoreLayers(self):
 
-        async with self.core.enterMigrationMode():
```

Comment:
All changes here are just indentation from no longer implementing the enterMigrationMode with block in here